### DISCUSSION TITLE
Problems publishing site

### DISCUSSION THREAD

**Discussion by rafadesign** [community/community#24025] [Other Feature Feedback, Questions, & Ideas]
<p>Hello<br>
I published a website on github pages, but it appears on this screen:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/e/3efc1350713dd69e872a9a866c3df3b5d65f8437.png" data-download-href="/uploads/short-url/8ZbFGoOtqNBiHl0fL7pWiwMV2uP.png?dl=1" title="Captura de tela de 2021-02-23 14-24-14" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/46974593/181080671-afa85d4f-c6f4-4602-a0d3-4d68b9f3dd29.png" alt="Captura de tela de 2021-02-23 14-24-14" data-base62-sha1="8ZbFGoOtqNBiHl0fL7pWiwMV2uP" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/e/3efc1350713dd69e872a9a866c3df3b5d65f8437_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/e/3efc1350713dd69e872a9a866c3df3b5d65f8437_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/e/3efc1350713dd69e872a9a866c3df3b5d65f8437_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/e/3efc1350713dd69e872a9a866c3df3b5d65f8437_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Captura de tela de 2021-02-23 14-24-14</span><span class="informations">1920×1080 55.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/rafadesign">@rafadesign</a>, welcome to the GitHub Support Community! GitHub provides a single <code>github.io</code> URL per account, using the account username. In your case yours would be <code>rafadesign.github.io</code>.</p>
<p>As you’ve named this repository <code>beefree.github.io</code> your site is online at the following URL:</p>
<p><a href="https://rafadesign.github.io/beefree.github.io/" class="onebox" target="_blank" rel="noopener">https://rafadesign.github.io/beefree.github.io/</a></p>

**Comment by rafadesign** [comment]
<p>got it, but your add a domain will look like this: <a href="https://rafadesign.github.io/beefreemarketing.com" rel="noopener nofollow ugc">https://rafadesign.github.io/beefreemarketing.com</a></p>
<p>or it will look like this: <a href="http://beefreemarketing.com" rel="noopener nofollow ugc">beefreemarketing.com</a><br>
?</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Oh yeah, if you add a custom domain it will appear directly at the domain you add, so <a href="https://beefreemarketing.com/">https://beefreemarketing.com/</a> :tada:</p>

**Comment by rafadesign** [comment]
<p>ok, thankyou very much</p>
